packages/react-examples/README.md

```diff
@@ -3,3 +3,71 @@
 **Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 
 This package is only meant to be consumed by variants of the Fluent UI documentation site and therefore has **NO GUARANTEE OF API STABILITY**.
+
+## Architecture
+
+This package contains various root folders within `src`
+
+```sh
+|- src/
+|--- e2e/
+|--- azure-themes/
+|--- react/
+|--- react-cards/
+|--- react-charting/
+|--- react-experiments/
+|--- react-focus/
+```
+
+1. `e2e/` contains helpers for e2e ? TODO
```

Comment:
Yes, though this can probably be deleted after Ling's PR converting to cypress component testing (all the utilities were for hacks to communicate between the test runner and story).